kubernetes/samples/charts/voyager/README.md

```diff
@@ -21,6 +26,23 @@ $ helm install appscode/voyager --name voyager-operator --version 7.4.0 \
   --set cloudProvider=baremetal \
   --set apiserver.enableValidatingWebhook=false
 ```
+Wait until Voyager Operator is running.
```

Comment:
Wait until Voyager Operator is running. -> Wait until the Voyager operator is running.